safehttp/response_writer.go

```diff
@@ -53,20 +53,67 @@ func NotWritten() Result {
 	return Result{}
 }
 
-// Write TODO
+// Write dispatches the response to the Dispatcher, setting the Content-Type to
+// text/html; charset=utf-8 and response status to 200 OK if resp is a safe
```

Comment:
This sets the `Content-Type` to whatever the dispatcher tells it to. Not necessarily `text/html`